src/agents/bee/runners/granite/prompts.ts

```diff
@@ -35,12 +42,22 @@ export const GraniteBeeSystemPrompt = new PromptTemplate({
         })
         .passthrough(),
     ),
+    createdAt: z.string().datetime().nullish(),
```

Comment:
Extend the default `BeeSystemPrompt` to `createdAt` (as this might be useful for others too).

Then, do the fork the following way:

```ts
BeeSystemPrompt.fork((config) => ({
   ...config,
   defaults: {
         ...config.defaults,
      instructions: '',
   },
   functions: {
      ...config.functions,
     formatDate: function() { ... }
   },
   template: `...`
}))
```

Reply:
Resolved in [a869350](https://github.com/i-am-bee/bee-agent-framework/pull/243/commits/a8693508f3ad4124a06541a39f551fe9d9cbdbbe), thanks for the tip.